modules/mono/editor/GodotTools/GodotTools.OpenVisualStudio/Program.cs

```diff
@@ -44,6 +44,21 @@ private static int Main(string[] args)
 
             var dte = FindInstanceEditingSolution(solutionFile);
 
+            if (dte == null && args.Length > 1 && args[1].Contains(";AddMethod"))
+            {
+                var stopwatch = new System.Diagnostics.Stopwatch();
+                stopwatch.Start();
```

Comment:
No need to use stopwatch here, just `DateTime.Now` and subtraction is enough precision; alternatively if you just do it for the code simplicity use the static `Stopwatch.StartNew()` method and remove the `Stop()` call (stopwatch does have any active components that actually get stopped)

Reply:
Changed to `DateTime.Now`